server/src/main/java/org/apache/calcite/avatica/server/HttpServer.java

```diff
@@ -274,7 +264,22 @@ protected void internalStart() {
     }
   }
 
-  private ServerConnector getConnector() {
+  protected void configureHandlers(ConstraintSecurityHandler securityHandler) {
```

Comment:
Is this type too narrow? Would there be other types of handlers that someone customizing the server would want to add to the list?